test/test_onnx.py

```diff
@@ -329,11 +329,10 @@ def test_paste_mask_in_image(self):
 
         assert torch.all(out2.eq(out_trace2))
 
-    @unittest.skip("Disable test until Resize opset 11 is implemented in ONNX Runtime")
     def test_mask_rcnn(self):
         images, test_images = self.get_test_images()
 
-        model = models.detection.mask_rcnn.maskrcnn_resnet50_fpn(pretrained=True)
+        model = models.detection.mask_rcnn.maskrcnn_resnet50_fpn(pretrained=True, min_size=200, max_size=300)
```

Comment:
For a follow-up PR, do you think it would be possible to remove the need for downloading the pre-trained weights in the tests?